packages/web-features/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "web-features",
   "description": "Curated list of Web platform features",
-  "version": "0.8.6",
+  "version": "0.8.7",
```

Comment:
I think the removal of `usage_stats` means this ought to be a minor version, `"0.9.0"`.

Reply:
You're right, I'll update.